pkg/clients/dynatrace/activegate_version_test.go

```diff
@@ -0,0 +1,29 @@
+package dynatrace
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func testActiveGateVersionGetLatestActiveGateVersion(t *testing.T, dynatraceClient Client) {
```

Comment:
```
TestActiveGateVersionGetLatestActiveGateVersion
```

Reply:
all of the dynatrace_client_test's are private, did not want to break the consistency

Reply:
https://github.com/Dynatrace/dynatrace-operator/pull/2604/files#diff-89f7ab64337128f7d4f99695e1982402ca568cbe6a189e26946ec16b1e4626a1R135